models/examples/example-1-repeat/PrepareRequest-FhirMessageUnsigned.json

```diff
@@ -127,8 +127,8 @@
           "coding": [
             {
               "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType",
-              "code": "acute",
-              "display": "Acute"
+              "code": "repeat-dispensing",
+              "display": "Repeat Dispensing"
             }
```

Comment:
Should these just be "repeat" for this example? Example is named example-1-repeat but payload has repeat dispensing. We have 'acute', 'repeat' and 'repeat-dispensing' in the value set for Prescription Type

Reply:
Good shout. I've used "repeat" to mean "repeat dispensing" here which is bound to cause confusion. Will rename the files.